its/pom.xml

```diff
@@ -20,7 +20,7 @@
   </organization>
 
   <properties>
-    <scannerMsbuild.version>5.15.0.80890</scannerMsbuild.version>
+    <scannerMsbuild.version>6.0.0.81631</scannerMsbuild.version>
```

Comment:
I think we should discuss the "how" of the current sprint before making these changes. One valid approach to avoid in the future this kind of commits is to always download the latest released version.